packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -246,6 +251,21 @@ try {
 	const primaryPhp = await requestHandler.getPrimaryPhp();
 	await apiEndpoint.setPrimaryPHP(primaryPhp);
 
+	// NOTE: We need to derive the loaded WP version or we might assume WP loaded
+	// from browser storage is the default version when it is actually something else.
+	// Incorrectly assuming WP version can break things like remote asset retrieval
+	// for minified WP builds.
+	const loadedWordPressVersion = await getLoadedWordPressVersion(
```

Comment:
Is this relevant for non-minified builds? Assets-wise, probably not, but it seems like it would enable displaying "6.6-nightly" in the UI, correct?

Reply:
> Is this relevant for non-minified builds? Assets-wise, probably not, but it seems like it would enable displaying "6.6-nightly" in the UI, correct?

Yes, I wasn't planning to do it as part of this PR, but we could show the detected version in the UI.